### PR TITLE
test: remove unused function in lua_integration_test.cc

### PR DESCRIPTION
**Pull request by crazyxy**
Signed-off-by: Yan Xue <yxyan@google.com>

Description:
Risk Level: low
Testing: bazel test //test/...
Docs Changes: N/A
Release Notes: N/A
[Optional Fixes #Issue]
[Optional Deprecated:]

### CODE REVIEW

File: test/extensions/filters/http/lua/lua_integration_test.cc [hunk 1]
```diff
@@ -431,7 +431,7 @@ name: envoy.lua
         request_handle:headers():add("signature_verification", "rejected")
         return
       end
-
+
```
Comment (zuercher) [inline test/extensions/filters/http/lua/lua_integration_test.cc]: nit: rm this please

File: test/extensions/filters/http/lua/lua_integration_test.cc [hunk 2]
```diff
@@ -445,7 +445,7 @@ name: envoy.lua
         request_handle:headers():add("signature_verification", "rejected")
       end
 
-      request_handle:releasePublicKey(pubkey)
+      request_handle:headers():add("verification", "done")
```
Comment (zuercher) [inline test/extensions/filters/http/lua/lua_integration_test.cc]: Was this just ignored before?

Comment (crazyxy) [reply test/extensions/filters/http/lua/lua_integration_test.cc]: Yes. It was ignored before even thought we didn't define `releasePublicKey`.

**Comment by zuercher** [comment]
Looks good to me, but I think you'll need to merge master to fix the release build error.

That said, CI is pretty wonky right now, so you might hold off until we've straightened out whatever billing issue we have with Circle CI.

**Pull request merged**